scripts/test.js

```diff
@@ -136,14 +136,28 @@ async function runTests() {
         cwd: path.join(__dirname, "..", "tests/docstrings_examples"),
         stdio: [0, 1, 2],
       });
-      // Ignore some tests not supported by node v18
-      // cp.execSync(
-      //   `node ${path.join("tests", "docstrings_examples", "DocTest.res.mjs")} --ignore-runtime-tests "Array.toReversed, Array.toSorted, Promise.withResolvers, Set.union, Set.isSupersetOf, Set.isSubsetOf, Set.isDisjointFrom, Set.intersection, Set.symmetricDifference, Set.difference"`,
-      //   {
-      //     cwd: path.join(__dirname, ".."),
-      //     stdio: [0, 1, 2],
-      //   },
-      // );
+
+      cp.execSync(`node ${path.join("tests", "docstrings_examples", "DocTest.res.mjs")}`, {
+        cwd: path.join(__dirname, ".."),
+        stdio: [0, 1, 2],
+      })
+
+      cp.execSync(`${rescript_exe} build`, {
+        cwd: path.join(__dirname, "..", "tests/docstrings_examples"),
+        stdio: [0, 1, 2],
+      });
```

Comment:
Run build again to check compile errors

Reply:
Could you add this as a comment in the code?

Reply:
Done